use force_* methods instead of bang methods

delba opened the pull request:
It should solve #67.

However, **this is not backward compatible** as the bang methods (`destroy!` and `delete!`) have been completely removed.

Alternatively, we could keep the bang methods for now and add a deprecation warning (?)

sohara committed:
Use quoted table name in default scope (call me paranoid)

radar committed:
Bump to 2.0.1

Commit:
use force_* methods instead of bang methods

radar commented:
Hi @delba, this PR does not merge correctly according to GH. Could you please fix that up?

I am all for adding a deprecation warning for those methods and keeping it in there for a month or two. After that, we can review and remove that warning and do another release.

johanneswuerbach committed:
Replaced deprecated gemcutter source

radar committed:
Add anti-regression test for #81

Update/save callbacks should not be called at all during a destroy call, as would happen with the code from that PR.

Commit:
Add configuration option for column name

References: https://github.com/goncalossilva/acts_as_paranoid

Fixes #76

Conflicts:
	lib/paranoia.rb

delba commented:
Ok, I just did something stupid. I rebased the original commit and then force pushed.
I will mess everything up so I guess I have to close this PR.
